Components/ListSearchResults/Partials/Header/_style.scss

```diff
@@ -1,59 +1,83 @@
+$search-form-spacing: 6px;
+
 .header {
   &-title {
-    @include reset-content-margins;
-
-    padding: $container-padding-desktop 0;
+    text-align: center;
   }
 }
 
 .searchForm {
+  position: relative;
+
   &-label {
-    height: 1px;
-    overflow: hidden;
+    left: 10px;
+    margin-bottom: 0;
     position: absolute;
-    width: 1px;
+    top: 50%;
+    transform: translateY(-50%);
+    z-index: 10;
+  }
+
+  &-icon {
+    color: $color-headline;
   }
 
   &-field {
+    background-color: #fff;
+    border: 1px solid #fff;
+    border-radius: 4px;
+    box-shadow: 0 1px 26px 0 rgba(0, 0, 0, 0.06);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: $search-form-spacing;
+    position: relative;
+    width: 100%;
+
+    @media (min-width: $breakpoint-mobile-horizontal) {
+      flex-direction: row;
+    }
+  }
+
+  &-box {
+    flex-grow: 1;
```

Comment:
How about instead of adding padding to the box, adding padding to the input instead? That way, I could imagine we could save some code and wouldn't need the autofill hack as well.

Reply:
Input does not actually fill in the entire "box" area (please see screesnshot). It was necessary to use this approach, since the width of the button might vary with different translations/content.

When the `autofill` styling kicks in, the `input` element gets a different color to the one of the box and looks a bit weird, in my opinion.

![search](https://user-images.githubusercontent.com/49150551/61526614-01a5e900-aa1b-11e9-8ffc-1589f116ccf4.png)

Reply:
Understand with the flex-grow. I would probably use `width: 100%` instead, but doesn't make a difference then. Just double checking: what's the `flex-shrink: 1` on the input doing?

In terms of the autofill, I would like to keep the browser functionality, as that's what people expect and are used to. There might be exceptions to the rule, but I feel making the spacing adjustments as suggested, we can still make it look nice and keep the browser default.

Reply:
I forgot to mention that, I've already removed the `flex-shrink: 1` from the `input`.

Will remove the autofill code and also implement the padding as suggested.